src/AutoRest.CSharp/LowLevel/Output/ConvenienceMethod.cs

```diff
@@ -21,17 +21,20 @@ internal record ConvenienceMethod(MethodSignature Signature, IReadOnlyList<Proto
     {
         public (IReadOnlyList<FormattableString> ParameterValues, Action<CodeWriter> Converter) GetParameterValues(CodeWriterDeclaration contextVariable)
         {
-            var (parameterValues, spreadVariable) = PrepareConvenienceMethodParameters(contextVariable);
+            var (parameterValues, contentInfo, spreadVariable) = PrepareConvenienceMethodParameters(contextVariable);
 
-            var converter = EnsureConvenienceBodyConverter(spreadVariable, contextVariable);
+            var converter = EnsureConvenienceBodyConverter(spreadVariable, contextVariable, contentInfo);
 
             return (parameterValues, converter);
         }
 
-        private (IReadOnlyList<FormattableString> ParameterValues, CodeWriterDeclaration? SpreadVariable) PrepareConvenienceMethodParameters(CodeWriterDeclaration contextVariable)
+        private record ContentInfo(CodeWriterDeclaration ContentVariable, FormattableString ContentValue);
```

Comment:
`ContentInfo` is a little confusing and we may lose the meaning of this struct. Maybe we can name it as `RequestContentParamterInfo`